src/test/java/com/conveyal/osmlib/PostgresTest.java

```diff
@@ -0,0 +1,58 @@
+package com.conveyal.osmlib;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.sql.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class PostgresTest {
```

Comment:
As a matter of policy all new classes and methods should have comments. This should contain a human-readable explanation of what is being tested.